assets/css/_logged_in_as.scss

```diff
@@ -0,0 +1,25 @@
+.c-logged-in-as__container {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.c-logged-in-as__icon {
+  height: 32px;
+  width: 32px;
+}
+
+.c-logged-in-as__text {
+  font-weight: 400;
+  font-size: 14px;
```

Comment:
**suggestion:** rather than hardcoding these you can use values defined in `_skate_ui.scss`.

Reply:
Ahhh cool - https://github.com/mbta/skate/pull/2383/commits/e40fec99962d5201d3358ef704403ebc094acb1c